editor/editor_settings.cpp

```diff
@@ -563,6 +563,10 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("text_editor/behavior/navigation/drag_and_drop_selection", true);
 	_initial_set("text_editor/behavior/navigation/stay_in_script_editor_on_node_selected", true);
 
+#ifdef LINUXBSD_ENABLED
+	EDITOR_SETTING(Variant::BOOL, PROPERTY_HINT_NONE, "text_editor/behavior/input/middle_mouse_button_paste", true, "")
+#endif
```

Comment:
Having it in `ifdef LINUXBSD_ENABLED` will likely mess up documentation updates (since different platforms will the different property list). There's also not much reason to restrict it to Linux only, this feature can be used on any platform (but should be disabled by default for the rest).

```suggestion
#ifdef LINUXBSD_ENABLED
	EDITOR_SETTING(Variant::BOOL, PROPERTY_HINT_NONE, "text_editor/behavior/input/middle_mouse_button_paste", true, "")
#else
	EDITOR_SETTING(Variant::BOOL, PROPERTY_HINT_NONE, "text_editor/behavior/input/middle_mouse_button_paste", false, "")
#endif
```

But `TextEdit` and `LineEdit` will need an additional check to use default or primary clipboard based on `FEATURE_CLIPBOARD_PRIMARY` flag to work on the outer platforms.

Reply:
Done!